src/frontend/src/flows/manage/settingsDropdown.ts

```diff
@@ -1,22 +1,42 @@
 import { dropdownIcon } from "$src/components/icons";
+import { handleKeyPress } from "$src/utils/keyboard.utils";
+import { Chan } from "$src/utils/utils";
 import { html, TemplateResult } from "lit-html";
+import { asyncReplace } from "lit-html/directives/async-replace.js";
 
-export const settingsDropdown = ({
-  alias,
-  id,
-  settings,
-}: {
+interface SettingsDropdown {
```

Comment:
In other modules we never use `interface`, and as much as possible try to keep then type/function signature inside the function where it's used. Would it make sense to do the same here?

```ts
type SettingsDropdown = Parameters<typeof mkChasm>[0];
```